### PR TITLE
minor doc fixes

### PR DESCRIPTION
**Pull request by aruediger**

### CODE REVIEW

**Comment by ktoso** [comment]
OK TO TEST

**Comment by ktoso** [comment]
Thanks a lot for helping out @2beaucoup!

File: akka-docs-dev/rst/scala/http/routing-dsl/testkit.rst [hunk 1]
```diff
@@ -102,6 +102,6 @@ Examples
 A great pool of examples are the tests for all the predefined directives in Akka HTTP.
 They can be found here__.
 
-__ @github@/akka-http-tests-scala/src/test/scala/akka/http/scaladsl/server/directives/
+__ @github@/akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/
```
Comment (ktoso) [inline akka-docs-dev/rst/scala/http/routing-dsl/testkit.rst]: great catch

**Comment by jrudolph** [comment]
Yes, thanks! LGTM. Hopefully we get all those through without conflicts.

**Comment by ktoso** [comment]
I'll take care of rebases etc if there need to be any, thanks everyone :-)

**Comment by 2beaucoup** [comment]
@jrudolph doesn't look too bad after a quick glance. (famous last words...) but this is reallly just minor stuff.

**Comment by ktoso** [comment]
LGTM, thanks :)

**Pull request merged**